test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/CustomSearchParamTests.cs

```diff
@@ -36,6 +36,7 @@ public async Task GivenANewSearchParam_WhenReindexingComplete_ThenResourcesSearc
             var patientName = Guid.NewGuid().ToString().ComputeHash().Substring(28).ToLower();
             var patient = new Patient { Name = new List<HumanName> { new HumanName { Family = patientName } } };
             var searchParam = Samples.GetJsonSample<SearchParameter>("SearchParameter");
+            searchParam.Code = "fooCode";
```

Comment:
>searchParam.Code = "fooCode"; [](start = 12, length = 29)

Just curios, why?
It's already have value:
`"code": "name"`
If it's confusing, why not change json itself?

Reply:
As this was a new change, I wanted to be explicit that it was being set.

Reply:
code Σ 1..1 code Code used in URL
I think it's defined to be always present, and if it's not, you can parse it into `SearchParameter`.

---
In reply to: [568172268](https://github.com/microsoft/fhir-server/pull/1629#discussion_r568172268) [](ancestors = 568172268)